PlayerControls/sources/AdVideoControls.swift

```diff
@@ -63,24 +63,25 @@ public final class AdVideoControls: UIViewController {
         airplayActiveView.isHidden = props.airplayActiveViewHidden
     }
     
-    public struct Props {
-        public static let `default` = Props(mainAction: .play { },
+    public struct Props: Codable {
+        public static let `default` = Props(mainCommand: .play(.nop),
                                             seeker: nil,
-                                            tapAction: nil,
+                                            tapCommand: nil,
                                             isLoading: true,
                                             airplayActiveViewHidden: true)
-        public let mainAction: MainAction
+        public let mainCommand: MainCommand
```

Comment:
Do we need to name `XxxxCommand` everything? I think using type inference it will be clear what is going on.

Reply:
Well, because `xxxAcion` becomes `xxxCommand` that contains action. It looked confusing for me to leave it as Action, when it was decieded to get rid of all of them.